packages/lesswrong/server/callbacks/postCallbacks.ts

```diff
@@ -212,6 +216,79 @@ getCollectionHooks("Posts").updateAsync.add(async function updatedPostMaybeTrigg
   }
 });
 
+interface SendPostRejectionPMParams {
+  messageContents: string,
+  lwAccount: DbUser,
+  post: DbPost,
+  noEmail: boolean,
+}
+
+async function sendPostRejectionPM({ messageContents, lwAccount, post, noEmail }: SendPostRejectionPMParams) {
+  const conversationData: CreateMutatorParams<DbConversation>['document'] = {
+    participantIds: [post.userId, lwAccount._id],
+    title: `Your post ${post.title} was rejected`,
+    moderator: true
+  };
+
+  const conversation = await createMutator({
+    collection: Conversations,
+    document: conversationData,
+    currentUser: lwAccount,
+    validate: false
+  });
+
+  const messageData = {
+    userId: lwAccount._id,
+    contents: {
+      originalContents: {
+        type: "html",
+        data: messageContents
+      }
+    },
+    conversationId: conversation.data._id,
+    noEmail: noEmail
+  };
+
+  await createMutator({
+    collection: Messages,
+    document: messageData,
+    currentUser: lwAccount,
+    validate: false
+  });
+
+  if (!isAnyTest) {
+    // eslint-disable-next-line no-console
+    console.log("Sent moderation message for post", post._id);
+  }
+}
+
+getCollectionHooks("Posts").updateAsync.add(async function sendRejectionPM({ newDocument: post, oldDocument: oldPost, currentUser }) {
+  const postRejected = post.rejected && !oldPost.rejected;
+  if (postRejected) {
+    const postUser = await Users.findOne({_id: post.userId});
+
+    let firstMessageContents =
+        // TODO: make link conditional on forum, or something
+        `Unfortunately, I rejected your post "${post.title}".  (The LessWrong moderator team is raising its moderation standards, see <a href="https://www.lesswrong.com/posts/kyDsgQGHoLkXz6vKL/lw-team-is-adjusting-moderation-policy">this announcement</a> for details).`
+  
+    if (post.rejectedReason) {
+      firstMessageContents += ` Your post didn't meet the bar for at least the following reason(s): ${post.rejectedReason}`;
+    }
+      
+    // EAForum always sends an email when deleting comments. Other ForumMagnum sites send emails if the user has been approved, but not otherwise (so that admins can reject comments by mediocre users without sending them an email notification that might draw their attention back to the site.)
```

Comment:
This copy-pasted comment and behavior maybe/maybe doesn't apply?

Reply:
I'd write:
// EA Forum Look Here. Decide if you want this to always send emails the way you do for deletion. We think it's better not to.